cmd/helm/install.go

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 package main
 
 import (
+	"bytes"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"html/template"
```

Comment:
I tried to switch to it and it doesn't work with the funcmaps. But, asides from that, is this roughly what you had in mind for the feature?